networkit/cpp/graph/Graph.cpp

```diff
@@ -678,11 +681,11 @@ void Graph::removeEdge(node u, node v) {
     }
     if (maintainCompactEdges) {
         // re-index edge IDs from deleted edge upwards
-        balancedParallelForNodes([&](node u) {
-            for (index i = 0; i < outEdges[u].size(); ++i) {
-                auto curID = outEdgeIds[u][i];
+        balancedParallelForNodes([&](node uu) {
```

Comment:
Good catch! The idiomatic way in NetworKit is to increment the char - better use `w` instead of `uu` here.